src/components/InfoBar/index.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import APP_STATES from '../../constants/APP_STATES';
+import SORT_BY from '../../constants/SORT_BY';
+
+class InfoBar extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {sortBy: SORT_BY.RELEASE_DATE}
+    }
+
+    onSortByReleaseDateClick = () => {
+        this.setState({
+            sortBy: SORT_BY.RELEASE_DATE
+        }, () => {
+            this.props.onSearch(this.state)
+        });
+    }
+
+    onSortByRatingClick = () =>{
+        this.setState({
+            sortBy: SORT_BY.RATING
+        }, () => {
+            this.props.onSearch(this.state)
+        });
+    }
+
+    render() {
+        let output = {};
```

Comment:
If switch doesn't handle any case, render will return ```{}```.
If you don't want render anything, you should return ```null``` (```false``` will work, but honestly I don't know how it's correct. ```undefined``` invokes an error).